packages/front-end/pages/features/index.tsx

```diff
@@ -48,6 +49,10 @@ export default function FeaturesPage() {
   const [currentPage, setCurrentPage] = useState(1);
   const [showArchived, setShowArchived] = useState(false);
   const [showSteps, setShowSteps] = useState(false);
+  const [
+    featureToDuplicate,
+    setFeatureToDupe,
```

Comment:
nit, probably better to use standard naming (setFeatureToDuplicate) for ease of finding in the codebase